Description: Add versioned soname to library
 Upstream has no versioned soname yet, so this patch adds a Debian
 specific soname version until something is added upstream.

---
--- a/makefile.in
+++ b/makefile.in
@@ -1221,7 +1221,7 @@
 	${AR} r $@ $^
 	${RANLIB} $@
 
-libliquid.so: libliquid.a
+$(SHARED_LIB): libliquid.a
 	$(CC) $(CFLAGS) $(LDFLAGS) -shared -Xlinker -soname=$@ -o $@ -Wl,-whole-archive $^ -Wl,-no-whole-archive $(LIBS)
 
 # static archive and library objects
@@ -1247,6 +1247,7 @@
 	mkdir -p $(DESTDIR)$(prefix)/include/liquid
 	mkdir -p $(DESTDIR)$(libdir)
 	install -m 644 -p ${ARCHIVE_LIB} ${SHARED_LIB} $(DESTDIR)$(libdir)
+	ln -s $(SHARED_LIB) $(DESTDIR)$(libdir)/libliquid.so
 	install -m 644 -p $(addprefix include/,$(headers_install)) $(DESTDIR)$(prefix)/include/liquid
 	@echo ""
 	@echo "---------------------------------------------------------"
--- a/configure.ac
+++ b/configure.ac
@@ -270,7 +270,7 @@
     AC_PROG_AR
 
     AR_LIB=libliquid.a
-    SH_LIB=libliquid.so
+    SH_LIB=libliquid.so.1
     REBIND=ldconfig
     ;;
 esac
